README.md

```diff
@@ -14,6 +14,7 @@ We support:
 
 * [OS X Mavericks (10.9)](https://itunes.apple.com/us/app/os-x-mavericks/id675248567)
```

Comment:
This link no longer works. I get an error saying "The item you requested is no longer available in the US itunes store".

Should we even link to where to find OS X versions? I assume people who really want an old version will have their own ways of finding them, and if Apple doesn't offer them we can't really help.